staging/src/k8s.io/apiextensions-apiserver/test/integration/subresources_test.go

```diff
@@ -322,6 +339,7 @@ func TestScaleSubresource(t *testing.T) {
 }
 
 func TestValidationSchemaWithStatus(t *testing.T) {
+	defer utilfeaturetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, apiextensionsfeatures.CustomResourceWebhookConversion, true)()
```

Comment:
nit: Not sure if this pattern is common in the test code (ignore if it is), but I think it is better to do something like 

```
featureCleanup := utilfeaturetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, apiextensionsfeatures.CustomResourceWebhookConversion, true)
defer featureCleanup()
```